vendor/wp-now/src/start-server.ts

```diff
@@ -87,6 +87,7 @@ export async function startServer(
 			new Promise((res) => {
 				server.close(() => {
 					output?.log(`Server stopped`);
+					php.exit();
```

Comment:
I've added the `php.exit()` call here to ensure we clean up the instance and that will trigger the event handler that e added for the symlink manager as well. @fluiddot